electron/firstRunDetector.js

```diff
@@ -1,17 +1,17 @@
 class FirstRunDetector {
-    constructor(firstRun){
-        this.firstRun = firstRun;
-    }
-    detectFirstRun = (event) => {
-        const isFirstRun = this.firstRun();
-        // Clear the firstRun setting, don't do this
-        //this.firstRun.clear();
+  constructor(firstRun) {
```

Comment:
I guess we should synchronize our whitespace settings :)

Reply:
Ah, I forgot to turn it off this team. I recommend using editconfig, eslint and prettier to solve this :-)